test/integration/smtpEmail.test.ts

```diff
@@ -130,3 +131,49 @@ describe('SMTP email', () => {
     })
   })
 })
+
+function setupSmtpTestEnvironment() {
```

Comment:
could move these to `/helpers/smtpEmails.ts` instead but they would seem to be misplaced there